tests/test_expense_tools.py

```
@@ -139,370 +108,218 @@ def test_list_expenses_with_filters(self, mock_api_request):
-            },
-            {
-                "account_id": "account123",
-                "amount": 200.00,
-                "description": "Printer ink"
-            }
-        ]
+        expense_date = date(2025, 1, 15)
         
-        # Call the function
-        result = create_expense(
+        result = await create_expense(
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)